### PR TITLE
New blog post - the business case for no business case

### PR DESCRIPTION
**Pull request by torotimes**
Hey - I'd really like to get someone to review this one - especially how I have placed the images. 

I haven't figured out how to run the yarn thingy!

### CODE REVIEW

**Review by jorunfa — COMMENTED** [review]
I'll make a new review of the actual content and try spinning it up to see how it looks, but here is some initial feedback.

File: blog/2022-05-24-no-need-for-a-business-case/index.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+To be a bit blunt, operators have in some ways chosen the worst of both worlds. They have chosen an outsourced model, but they’re not taking advantage of it. Technology vendors are essentially told how to solve the problem. 
+
+Instead, operators should give them a service-level agreement (SLA) with what they want to achieve and then let the vendor figure out how to solve it. While the rest of the world has figured out when to do things in-house and when to outsource, mobile operators are often stuck in the middle. 
+
+**What’s the magic pill?** Micro innovation at scale. How do you make it happen? In the words of Appledore Research, by understanding that as-a-service is more than a simple substitution of capital costs with an operational one - and more than replacing a one-time expense with an amortized one. 
+
+Here’s how Robert Curran at Appledore Research broke it down in his recent report on the significance of the deal between Working Group Two and CKH IOD. 
+
+![](./bizcase.png)
```
[the remark this answers is not in the document]
Comment (torotimes) [reply blog/2022-05-24-no-need-for-a-business-case/index.md:50]: |                        |                                     **_Managed Service _**                                    |                                        **_As-a-Service_**                                        |
|:----------------------:|:---------------------------------------------------------------------------------------------:|:------------------------------------------------------------------------------------------------:|
|     **Deployment **    |                                       Customer specific                                       |                               Common platform used by all customers                              |
|      **Support **      |                                    Customer-specific team                                     |                                            Shared team                                           |
| **Change Management ** |              Contractual negotiation – change  Requests and specification cycles              |                   Frequent platform updates, constant  feedback from customers                   |
|        **SLA **        |    Process-defined – requires extensive list of activities to be defined, targets and costs   |                Service-defined – availability and quality of the service provided                |
|      **Contract **     | High upfront cost to define scope and  terms; difficult and expensive to change  subsequently |                  Standardized contract terms, based on simple to understand SLAs                 |
|   **Business Model **  |                         Fixed cost envelope, no incentive to  innovate                        | Usage-based model, incentivized to make changes that improve customer experience/service quality |
|        **R&D **        |                             No dedicated resources to support R&D                             |                       Direct flow through from core R&D to in-use platform                       |

[the remark this answers is not in the document]
Comment (torotimes) [reply blog/2022-05-24-no-need-for-a-business-case/index.md:50]: |                        |                                      **Managed Service**                                      |                                         **As-a-Service**                                         |
|:----------------------:|:---------------------------------------------------------------------------------------------:|:------------------------------------------------------------------------------------------------:|
|     **Deployment **    |                                       Customer specific                                       |                               Common platform used by all customers                              |
|      **Support **      |                                    Customer-specific team                                     |                                            Shared team                                           |
| **Change Management ** |              Contractual negotiation – change  Requests and specification cycles              |                   Frequent platform updates, constant  feedback from customers                   |
|        **SLA **        |    Process-defined – requires extensive list of activities to be defined, targets and costs   |                Service-defined – availability and quality of the service provided                |
|      **Contract **     | High upfront cost to define scope and  terms; difficult and expensive to change  subsequently |                  Standardized contract terms, based on simple to understand SLAs                 |
|   **Business Model **  |                         Fixed cost envelope, no incentive to  innovate                        | Usage-based model, incentivized to make changes that improve customer experience/service quality |
|        **R&D **        |                             No dedicated resources to support R&D                             |                       Direct flow through from core R&D to in-use platform                       |

File: website/personio-jobs.xml [hunk 2]
```diff
@@ -3,55 +3,6 @@
 <workzag-jobs>
 
     
-        <position>
```
[the remark this answers is not in the document]
Comment (jorunfa) [reply website/personio-jobs.xml:6]: This change is in this PR for some reason. I can revert it.

File: blog/2022-05-24-no-need-for-a-business-case/index.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (torotimes) [reply blog/2022-05-24-no-need-for-a-business-case/index.md:50]: |                       |                                      **Managed Service**                                      |                                         **As-a-Service**                                         |
|:---------------------:|:---------------------------------------------------------------------------------------------:|:------------------------------------------------------------------------------------------------:|
|     **Deployment**    |                                       Customer specific                                       |                               Common platform used by all customers                              |
|      **Support**      |                                    Customer-specific team                                     |                                            Shared team                                           |
| **Change Management** |              Contractual negotiation – change  Requests and specification cycles              |                   Frequent platform updates, constant  feedback from customers                   |
|        **SLA**        |    Process-defined – requires extensive list of activities to be defined, targets and costs   |                Service-defined – availability and quality of the service provided                |
|      **Contract**     | High upfront cost to define scope and  terms; difficult and expensive to change  subsequently |                  Standardized contract terms, based on simple to understand SLAs                 |
|   **Business Model**  |                         Fixed cost envelope, no incentive to  innovate                        | Usage-based model, incentivized to make changes that improve customer experience/service quality |
|        **R&D**        |                             No dedicated resources to support R&D                             |                       Direct flow through from core R&D to in-use platform                       |

File: blog/2022-05-24-no-need-for-a-business-case/index.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
+---
+slug: no-need-for-a-business-case
+title: "The business case for not needing a business case"
+date: 2022-06-09
```
Comment (jorunfa) [inline blog/2022-05-24-no-need-for-a-business-case/index.md:4]: I'll change the filename to match this.

File: website/personio-jobs.xml (hunk 2 above)
[the remark this answers is not in the document]
Comment (jorunfa) [reply website/personio-jobs.xml]: Done.

File: blog/2022-06-09-no-need-for-a-business-case/index.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+slug: no-need-for-a-business-case
+title: "The business case for not needing a business case"
+date: 2022-06-09
+tags: [saas, operators, managed services,]
+authors: [erlend-prestgard]
+---
+
+You’ve heard these terms a million times: *Managed services* and *software-as-a-service*. At first glance, they appear deceptively similar. You wonder, how are they different again? Understanding the difference between them unlocks a perspective on where the telecom industry is heading. 
+
+![](./RFP_illustration_large.jpg)
```
Comment (jorunfa) [inline blog/2022-06-09-no-need-for-a-business-case/index.md:11]: The images should preferably have alt text for those with screen readers.

**Pull request merged**